Dockerfile

```diff
@@ -1,39 +1,33 @@
 FROM python:3.5
 RUN apt-get -y update && apt-get -y upgrade
-
 RUN apt-get install -y libpq-dev supervisor curl vim
```

Comment:
we should consider removing ```vim```. Nice for debugging containers, but not useful for the runtime environemnt itself.

Reply:
I prefer to keep `vim`. Sometimes it is required to do some hot fixes in running rkt containers. And this dockerfile are used for old rkt solution only.